src/lib/utils/resizeBase64Image.native.js

```diff
@@ -0,0 +1,8 @@
+import ImageResizer from 'react-native-image-resizer'
+import RNFS from 'react-native-fs'
+
+export default async (base64, sizeByWidth) => {
```

Comment:
1. the crop component already resizes images according to its settings no?
2. are we calling this method on native anywhere? or this is just to be future compatible?

Reply:
Yeah, we are, at src/lib/gundb/UserStorageClass.js:597. It's used for the smallAvatar